packages/core/src/components/tag/_common.scss

```diff
@@ -59,84 +48,66 @@ $tag-padding-large: ($tag-height-large - $pt-icon-size-large) / 2 !default;
     border-radius: $tag-height-large / 2;
   }
 
-  #{$removable-selector} {
+  &.pt-tag-removable {
     padding-right: $tag-height-large;
   }
 }
 
-@mixin tag-minimal() {
+@mixin pt-tag-intent($background-color, $text-color: $white) {
+  background: $background-color;
+  color: $text-color;
+}
+
+@mixin pt-tag-minimal() {
   &:not([class*="pt-intent-"]) {
-    background: rgba($gray3, 0.2);
+    background-color: rgba($gray3, 0.2);
```

Comment:
Out of scope for this PR, but whoa, didn't realize we were using a non-standard color for minimal-tag backgrounds. Reasoning for not just using `$light-gray4`?